app/services/moderator/merge_user.rb

```diff
@@ -31,17 +43,13 @@ def merge
     private
 
     def handle_identities
-      error_message = "The user being deleted already has two identities. " \
-                      "Are you sure this is the right user to be deleted? " \
-                      "If so, a super admin will need to do this from the console to be safe."
-      raise error_message if @delete_user.identities.count.positive?
+      raise MULTIPLE_IDENTITIES_ERROR_MSG if @delete_user.identities.count >= 2
+      raise DUPLICATE_IDENTITIES_ERROR_MSG if
+        (@delete_user.identities.pluck(:provider) & @keep_user.identities.pluck(:provider)).any?
 
-      return true if
-        @keep_user.identities.count.positive? ||
-          @delete_user.identities.none? ||
-          @keep_user.identities.last.provider == @delete_user.identities.last.provider
+      return true if @delete_user.identities.none?
 
-      @delete_user.identities.first.update_columns(user_id: @keep_user.id)
+      @delete_user.identities.update_all(user_id: @keep_user.id)
```

Comment:
Apparently `update_columns` doesn't work on ActiveRecord collections 🤔 I think this may have been broken, but it seems like merging users have been working fine outside of this fix.

Reply:
It was `.first.update_columns` so it was working on a single AR object, not a collection :)